src/Core/src/Fonts/FontManager.Android.cs

```diff
@@ -11,7 +12,6 @@ public class FontManager : IFontManager
 	{
 		static readonly string[] FontFolders = new[]
 		{
-			"",
```

Comment:
We are already attempting to use the paths as-is, so we don't need `""` as a specific directory to find fonts in.